lib/engine_socketio.js

```diff
@@ -180,52 +181,55 @@ SocketIoEngine.prototype.step = function (requestSpec, ee) {
   };
 
   function preStep(context, callback){
-    var namespace = requestSpec.emit.namespace || "/";
-    self.getSocket(namespace, context, function(err, socket){
+    // Set default namespace in emit action
+    requestSpec.emit.namespace = requestSpec.emit.namespace || "/";
+
+    self.loadContextSocket(requestSpec.emit.namespace, context, function(err, socket){
       if(err) {
         return callback(err, context);
       }
 
-      return f(context, callback, socket);
+      return f(context, callback);
     });
   }
 
   if(requestSpec.emit) {
+
     return preStep;
   } else {
     return f;
   }
 };
 
-SocketIoEngine.prototype.getSocket = function(namespace, context, cb) {
-  context.namespaces = context.namespaces || {};
+SocketIoEngine.prototype.loadContextSocket = function(namespace, context, cb) {
+  context.sockets = context.sockets || {};
```

Comment:
Renamed `context.namespaces` to `context.sockets`. Naming seemed more explanatory.